src/components/Article.jsx

```diff
@@ -1 +1,32 @@
-// export const Article = () => ();
+import React from 'react';
+
+export function Article({
+  title,
+  date,
+  text,
+}) {
+  const dateISO = date.toISOString().slice(0, 10);
+  const dateLocaleString = date.toLocaleDateString('en-US', {
+    year: 'numeric',
+    month: 'long',
+    day: 'numeric',
+  });
+
+  return (
+    <>
+      <h2 className="title" data-cy="title">
+        {title}
+      </h2>
+
+      <p className="subtitle">
+        <time dateTime={dateISO} data-cy="date">
+          {dateLocaleString}
+        </time>
+      </p>
+
+      <p data-cy="text">
+        {text}
+      </p>
+    </>
```

Comment:
```suggestion
    <div>
      <h2 className="title" data-cy="title">
        {title}
      </h2>

      <p className="subtitle">
        <time dateTime={dateISO} data-cy="date">
          {dateLocaleString}
        </time>
      </p>

      <p data-cy="text">
        {text}
      </p>
    </div>
```

Reply:
your component must have some container after it is unwrapped. React.fragment disappears after the component is unwrapped.